### DISCUSSION TITLE
Podman quadlet ipv6

### DISCUSSION THREAD

**Discussion by fabiopicchi** [dani-garcia/vaultwarden#4998] [General]
I did my best to expose vaultwarden to the internet using podman, but to no avail. I can curl my pod with its ipv6 address, but I can't expose it. Furthermore, I don't fully understand podman networking (seems to be a pretty complex topic) so I was wondering if anyone was able to get a similar setup working.

**Comment by stefan0xC** [comment]
Are you using a reverse proxy? Did you set `ROCKET_ADDRESS=::0`? To my understanding you have to enable ipv6 in the podman network for the firewall rules to be setup properly. Cf. this discussion https://github.com/containers/podman/issues/7415#issuecomment-986567501 and [this more recent article on how to enable ipv6 for podman 4.0](https://developers.redhat.com/articles/2022/08/10/how-conifgure-podman-40-ipv6)

**Reply by fabiopicchi** [reply]
Thanks for the swift reply!
Yes, I did set `ROCKET_ADDRESS=::0`. I am trying to avoid a reverse proxy for a simpler setup. I actually managed to get things working with docker (works out of the box with minimal setup).

I'll look into the links you provided, but I imagine this to be firewall related. I still struggle to understand how networking works with podman.

**Reply by BlackDex** [reply]
It shouldn't be `::0` it should be `::` without any digit. That is the equivalent of `0.0.0.0`

**Reply by BlackDex** [reply]
Also as a reference `::1` is the same as `127.0.0.1`

**Reply by stefan0xC** [reply]
> It shouldn't be `::0` it should be `::` without any digit. That is the equivalent of `0.0.0.0`

I think `::` is equivalent `0:0:0:0:0:0:0:0` which should be functionally the same as `::0` but it's probably nicer to leave it out. (btw: I've copied the answer from you https://github.com/dani-garcia/vaultwarden/discussions/3306#discussioncomment-5203940 😔 )
